server/py_files/models/LSTM/LSTMClassifier.py

```diff
@@ -18,18 +18,24 @@ def __init__(self, name, feature_type):
 
     def train(self, samples, labels):
         features = self.choose_features(samples, True)
+        model = Sequential()
 
-        if(self.feature_type != 'word-embeddings'):
-            raise Exception('LSTM model is only configured for word-embeddings at the moment please train wiht word embeddings')
-        #not sure we can use this max length if the length is diff than trained model is expecting then it will crash
-        # max_length = max([len(s) for s in samples]) # todo think: might be a costly step if huge data, may be it should just be a constant (100)
-        x_train = pad_sequences(features, maxlen=100, padding='post')
-
-        embedding_vecs = Embeddings(self.name, 100).vectors()
-        vocab_size = embedding_vecs.shape[0]
+        embedding_size = 100
+        if self.feature_type == 'word-embeddings':
+            # not sure we can use this max length if the length is diff than trained model is expecting then it will crash
+            # max_length = max([len(s) for s in samples]) # todo think: might be a costly step if huge data, may be it should just be a constant (100)
+            max_length = 100
+            x_train = pad_sequences(features, maxlen=max_length, padding='post')
+            pretrained_embeddings = Embeddings(self.name, embedding_size).vectors()
+            vocab_size = pretrained_embeddings.shape[0]
+            model.add(Embedding(input_dim=vocab_size, output_dim=embedding_size, weights=[pretrained_embeddings]))
+        elif self.feature_type in ['tf-idf', 'bow']:
```

Comment:
Cc @astrebeck (have not ported this for NN and CNN yet) - leaving to u

Note1: have moved the `choose_features()` out of the PythonServer and given that control to the model classes (so they can choose effectively)
Note2: the CNN doesn't seem to be working on my end - but didn't look much into it